packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.scss

```diff
@@ -0,0 +1,207 @@
+@import '../../common/common';
```

Comment:
These Persona file changes can be ignored, it's just leftovers from a bad copy-paste merge I'm having to do to split up the PRs